neo/Prompt/Utils.py

```diff
@@ -131,6 +131,38 @@ def get_from_addr(params):
     return params, from_addr
 
 
+def get_change_addr(params):
+    to_remove = []
+    change_addr = None
+    for item in params:
+        if '--change-addr=' in item:
+            to_remove.append(item)
+            try:
+                change_addr = item.replace('--change-addr=', '')
```

Comment:
if the return of this functions expects at best 1 `change_addr` then we do not need the `to_remove` list and loop over it. We can  do something like:
```
for item in params:
   change_addr = None
   if '--change-addr' in item:
      params.remove(item)
      change_addr = item.replace('--change-addr=','')
      return params, change_addr
```
This way we stop the moment we found the a change address and avoid unnecessary looping over the remaining params. 

What is the `Try/except` catching exactly? It's generally bad practice to use `Exception` (==catch all) and we should prefer being more specific (like `ValueError`or `TypeError`)

Reply:
The format for `get_change_addr` followed the example of the pre-existing `get_from_addr`. I will work on improving these utility functions.